test/common/quic/active_quic_listener_test.cc

```diff
@@ -525,14 +525,18 @@ TEST_P(ActiveQuicListenerTest, ProcessBufferedChlos) {
   for (size_t i = 1; i <= count; ++i) {
     sendCHLO(quic::test::TestConnectionId(i));
   }
-  dispatcher_->run(Event::Dispatcher::RunType::Block);
-  dispatcher_->run(Event::Dispatcher::RunType::NonBlock);
-  dispatcher_->run(Event::Dispatcher::RunType::NonBlock);
+  absl::SleepFor(absl::Milliseconds(1));
```

Comment:
The packets are being stacked in kernel recvQ during the waiting. There is no Envoy stats for this. sleep here is not necessary because the following expectations are adjusted according to whether all packets are read within 3 event loops or not. I added sleep here just to increase the chance of testing the desired behavior where all packets arrive at once.